alignment/sequencealigner_test.py

```diff
@@ -0,0 +1,254 @@
+"""
```

Comment:
Thanks a lot for this. I have objections about the new additions to the library. So, what would you say about keeping only the tests about the existing modules and leaving the rest for a future PR?